frontends/main/src/app-pages/HomePage/HomePage.tsx

```diff
@@ -43,24 +43,28 @@ const HomePage: React.FC = () => {
     <>
       <LearningResourceDrawer />
       <FullWidthBackground>
-        <Container>
+        <Container style={{ overflow: "hidden" }}>
```

Comment:
Fixes this issue where the hero image pushes past the container causing horizontal scroll:

<img width="1289" alt="image" src="https://github.com/user-attachments/assets/4bd7351a-c1fa-4974-a02d-88ea1b590e99">

Reply:
Ignore, addressed in https://github.com/mitodl/mit-learn/pull/1777